packages/concerto-cto/lib/parser.pegjs

```diff
@@ -1711,22 +1712,43 @@ ImportType
   }
 
 ImportTypes
-    = ImportToken __ ns:QualifiedNamespaceDeclaration ".{" _ types:commaSeparatedIdentifiers _ "}" __ u:FromUri? {
+    = ImportToken __ ns:QualifiedNamespaceDeclaration ".{" _ types:commaSeparatedTypes _ "}" __ u:FromUri? {
+    	const { aliasedTypes, remainingTypes } = types.reduce((acc, type) => {
+          if (type.$class === "concerto.metamodel@1.0.0.AliasType") {
+            acc.aliasedTypes.push(type);
+            acc.remainingTypes.push(type.name);
```

Comment:
```suggestion
            acc.typeNames.push(type.name);
```
`remainingTypes` threw me off, I had to read the code to understand that they are not actually the remaining types.

or

```suggestion
            acc.types.push(type.name);
```
to keep it aligned with what will be the final field.